data_steward/tools/load_submission.py

```diff
@@ -0,0 +1,124 @@
+    for blob in blobs:
+        table_name = _filename_to_table_name(blob.name)
+        if table_name not in AOU_REQUIRED:
+            continue
+        schema = get_table_schema(table_name)
+        hpo_table_name = f'{hpo_id}_{_filename_to_table_name(blob.name)}'
+        fq_hpo_table = f'{bq_client.project}.{dst_dataset}.{hpo_table_name}'
+        destination = Table(fq_hpo_table, schema=schema)
+        destination = bq_client.create_table(destination, exists_ok=True)
+        job_config = LoadJobConfig()
+        job_config.schema = schema
+        job_config.skip_leading_rows = 1
+        job_config.source_format = 'CSV'
+        source_uri = f'gs://{bucket_name}/{blob.name}'
+        load_job = bq_client.load_table_from_uri(source_uri,
+                                                 destination,
+                                                 job_config=job_config)
+        LOGGER.info(f'table:{destination} job_id:{load_job.job_id}')
+        load_jobs.append(load_job)
+        load_job.result()
+    return load_jobs
+
+
+def get_arg_parser():
+    argument_parser = argparse.ArgumentParser(description=__doc__)
+    argument_parser.add_argument(
+        '-p',
+        '--project_id',
+        dest='project_id',
+        action='store',
+        help='Identifies the project containing the target dataset',
+        required=True)
+    argument_parser.add_argument('-d',
+                                 '--dataset_id',
+                                 dest='dataset_id',
+                                 action='store',
+                                 help='Identifies the target dataset',
+                                 required=True)
+    argument_parser.add_argument('-b',
+                                 '--bucket_name',
+                                 dest='bucket_name',
+                                 action='store',
+                                 help='Bucket containing archive folders',
+                                 required=True)
+    argument_parser.add_argument('-i',
+                                 '--hpo_id',
+                                 dest='hpo_id',
+                                 action='store',
+                                 help='Identifies the hpo_id of the site',
+                                 required=True)
+    argument_parser.add_argument('-f',
+                                 '--folder_name',
+                                 dest='folder_name',
+                                 action='store',
+                                 help='Name of the submission folder to load',
+                                 required=True)
+    return argument_parser
+
+
+def main(project_id, dataset_id, bucket_name, hpo_id, folder_name):
+    bq_client = get_client(project_id)
+    gcs_client = Client(project_id)
+    site_bucket = get_bucket(bq_client, hpo_id)
+    prefix = f'{hpo_id}/{site_bucket}/{folder_name}'
+    load_stage(dataset_id, bq_client, bucket_name, prefix, gcs_client, hpo_id)
```

Comment:
nit:  It would be nice to log the procedure starting and finishing successfully.